dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -446,28 +449,44 @@ def test_null_count(self):
 
 class TestProfilerNullValues(unittest.TestCase):
 
-    @classmethod
-    def setUpClass(cls):
+    def test_correct_rows_ingested(self):
         test_dict = {
             '1': ['nan', 'null', None, None, ''],
             1: ['nan', 'None', 'null', None, ''],
         }
         test_dataset = pd.DataFrame(data=test_dict)
-        cls.trained_schema = dp.Profiler(test_dataset, len(test_dataset))
-
-    def test_correct_rows_ingested(self):
+        trained_schema = dp.Profiler(test_dataset, len(test_dataset))
+        
         self.assertCountEqual(['', 'nan', 'None', 'null'],
-                         self.trained_schema.profile['1'].null_types)
-        self.assertEqual(
-            5, self.trained_schema.profile['1'].null_count)
+                         trained_schema.profile['1'].null_types)
+        self.assertEqual(5, trained_schema.profile['1'].null_count)
         self.assertEqual({'': [4], 'nan': [0], 'None': [2, 3], 'null': [
-                         1]}, self.trained_schema.profile['1'].null_types_index)
+                         1]}, trained_schema.profile['1'].null_types_index)
         self.assertCountEqual(['', 'nan', 'None', 'null'],
-                         self.trained_schema.profile[1].null_types)
-        self.assertEqual(
-            5, self.trained_schema.profile[1].null_count)
+                         trained_schema.profile[1].null_types)
+        self.assertEqual(5, trained_schema.profile[1].null_count)
         self.assertEqual({'': [4], 'nan': [0], 'None': [1, 3], 'null': [
-                         2]}, self.trained_schema.profile[1].null_types_index)
+                         2]}, trained_schema.profile[1].null_types_index)
+
+    def test_correct_null_row_counts(self):
+        file_path = os.path.join(test_root_path, 'data', 'csv/empty_rows.txt')
+        data = pd.read_csv(file_path)
+        profiler_options = ProfilerOptions()
+        profiler_options.set({'data_labeler.is_enabled': False})
+        profile = dp.Profiler(data, profiler_options=profiler_options)
+        self.assertEqual(2, profile.row_has_null_count)
+        self.assertEqual(0.25, profile._get_row_has_null_ratio())
+        self.assertEqual(2, profile.row_is_null_count)
+        self.assertEqual(0.25, profile._get_row_is_null_ratio())
+        
+        
+        file_path = os.path.join(test_root_path, 'data','csv/iris-with-null-rows.csv')
+        data = pd.read_csv(file_path)
+        profile = dp.Profiler(data, profiler_options=profiler_options)
+        self.assertEqual(15, profile.row_has_null_count)
+        self.assertAlmostEqual(0.1, profile._get_row_has_null_ratio())
+        self.assertEqual(7, profile.row_is_null_count)
+        self.assertAlmostEqual(0.04666666, profile._get_row_is_null_ratio())
```

Comment:
if we know the fraction, which I believe we should, we could use the fraction instead of `assertAlmostEqual`